src/content/docs/browser/new-relic-browser/browser-pro-features/browser-features.mdx

```diff
@@ -11,6 +11,7 @@ Check out our key <InlinePopover type="browser" /> features:
 
```

Comment:
Just a general comment that this is a pretty odd doc. I don't think it should be fixed/addressed in this PR, but I'm wondering if there's a better way to present these core browser monitoring features. 

Also, related to accessibility, making everything link text is pretty tough for screen readers. Again, not something that should be addressed in this PR, but something to keep in mind in future.

Reply:
Agreed. I can a create another ticket to address this page.